### DISCUSSION TITLE
Commendation

### DISCUSSION THREAD

**Discussion by taqisyed** [IchthysMaranatha/asterisk-chan-quectel#10] [General]
First congrats on setting this project up.
I successfully setup Sim7600 module with raspberry pi zero. Running asterisk 13 and Freepbx GUI trunk, outbound & inbound routes are set and all is going well. 
Thanks for this,
Great Job.

**Comment by IchthysMaranatha** [comment]
Thank you for the feedback.

**Comment by 1iri** [comment]
> First congrats on setting this project up. I successfully setup Sim7600 module with raspberry pi zero. Running asterisk 13 and Freepbx GUI trunk, outbound & inbound routes are set and all is going well. Thanks for this, Great Job.

Hello,
Indeed, thank you very much for this project, I'm able to run quectel with on sim7600 with voice support, much appreciated!

[taqisyed](https://github.com/taqisyed), Do you happen to have a quick tutorial for what you've achieved, I'm trying the same thing on a Pi4 with Sim7600, with FreePBX GUI, I was able to get it going with a Huawei modem and chan_dongle with raspbx, setting the trunk, outbound & inbound routes in FreePBX and everything was good. But I'm struggling with quectel. Probably I'm using the dialplan completely wrong, or maybe I'm not setting everything the right way in FreePBX.

Can you help with some basic configs you did in FreePBX gui?

Thanks!
